mediapipe/tasks/ios/vision/image_segmenter/sources/MPPImageSegmenter.h

```diff
@@ -203,7 +203,7 @@ NS_SWIFT_NAME(ImageSegmenter)
  *
  * @return `YES` if the image was sent to the task successfully, otherwise `NO`.
  */
-- (BOOL)segmentAsyncInImage:(MPPImage *)image
+- (BOOL)segmentAsyncImage:(MPPImage *)image
```

Comment:
Since we tend to keep method names like sentences in ObjC, I went by meaning. Like we usually say classify image, segment image but detect objects in image. But looking back, gesture recogniser's should have been inImage and not image. 

To keep it consistent wouldn't it be better to drop the in everywhere?  Styliser: stylise and embedder: embed, Generator: generate. In for these will sound off. 
This would mean modifying the detector and landmarkers.

Reply:
I do prefer `segmentAsyncImage`, but only because it aligns better with the other APIs (`segmentImage`, `segmentVideoFrame`.

Let's get an opinion from @khanhlvg since I feel a bit uneasy about breaking our API for almost all tasks (but this is probably not an API that is actively used since most users will go through Swift).

Reply:
If we need to change API name for consistency then we should do it now rather than later because we've only announced the iOS SDK a month ago so adoption is still small.

I'm fine with breaking API changes now to make them all consistent.

Reply:
Sounds good. Let's change the API for all tasks in a follow up.